src/drawerbase.js

```diff
@@ -368,7 +368,7 @@ OpenSeadragon.DrawerBase = class DrawerBase{
         * @property {OpenSeadragon.DrawerBase} drawer - The drawer that raised the error.
         * @property {String} error - A message describing the error.
         * @property {?Object} userData - Arbitrary subscriber-defined object.
-        * @private
+        * @protected
```

Comment:
What do you mean by this? We're using `private` to mark functions (and events) that aren't part of the API, even if they are used by other classes. So I guess `protected` would mean something that is an official part of the API but should only be used by subclasses of DrawerBase? Seems like that might not be a terribly important distinction to make, since we don't have any sort of way to enforce it. Either it's part of the API or it's not.

Reply:
My IDE complainted because drawers are (from the look of current implementation) supposed to call this method to say 'error when rendering'. If it is marked private, it might people lead to think it is not to be used. At least that's the cuse from the current webgl implementation.

Reply:
Fair enough. I suppose it's fine to leave like this.